app/models/move.rb

```diff
@@ -2,8 +2,9 @@ class Move < ApplicationRecord
   belongs_to :game
   belongs_to :piece
 
+  # If even, black is up.  If odd, white.
   def turn
```

Comment:
Would it be possible to move this to the Game model?  Since most likely we'd need to query the game for who's turn it is.  Should be possible to just copy/paste and then remove `game.` from everything inside the method.